packages/manager/modules/server-sidebar/src/web.constants.js

```diff
@@ -165,6 +166,17 @@ export const EMAIL_CONFIG = {
       icon: 'ovh-font ovh-font-mail',
       app: [WEB],
       feature: 'emails:mxplan',
+      filter: {
+        category: 'mxplan',
+        fn: (items, compareTo) =>
+          items.filter((it) =>
+            isString(
+              find(get(compareTo, 'domain', []), {
+                displayName: it.displayName,
+              }),
+            ),
+          ),
```

Comment:
There is too much nested lodash functions here, please use native javascript when possible
```suggestion
          items.filter(({ displayName }) =>
            typeof compareTo.domain?.find((domain) =>
              domain.displayName === displayName,
            ) === 'string',
          ),
```

Reply:
it's corrected